contracts/deploy/01-foreign-chain.ts

```diff
@@ -56,20 +59,29 @@ const deployForeignGateway: DeployFunction = async (hre: HardhatRuntimeEnvironme
     nonce = await homeChainProvider.getTransactionCount(deployer);
     nonce += 1; // HomeGatewayToEthereum deploy tx will the third tx after this on its home network, so we add two to the current nonce.
   }
-  const { claimDeposit, challengeDuration, homeChainId } = paramsByChainId[chainId];
+  const { claimDeposit, challengeDuration, homeChainId, arbitrumInbox } = paramsByChainId[chainId];
   const challengeDeposit = claimDeposit;
   const bridgeAlpha = 5000;
   const homeChainIdAsBytes32 = hexZeroPad(homeChainId, 32);
 
   const homeGatewayAddress = getContractAddress(deployer, nonce);
   console.log("calculated future HomeGatewayToEthereum address for nonce %d: %s", nonce, homeGatewayAddress);
 
+  nonce -= 1;
+
+  const fastBridgeSenderAddress = getContractAddress(deployer, nonce);
+  console.log("calculated future FastSender for nonce %d: %s", nonce, fastBridgeSenderAddress);
+
+  nonce += 5;
+  const inboxAddress = chainId === ForeignChains.HARDHAT ? getContractAddress(deployer, nonce) : arbitrumInbox;
```

Comment:
For live networks like Rinkeby/Mainnet, we don't deploy any `Inbox` mock contract so `getContractAddress()` would fail. We need to use the Inbox addresses provided by Arbitrum instead.